frontend/apps/reader/modules/readerview.lua

```diff
@@ -164,7 +164,7 @@ function ReaderView:paintTo(bb, x, y)
     end
 
     -- draw page content
-    if self.ui.document.info.has_pages then
+    if self.document.info.has_pages then
```

Comment:
The new trend is to replace `if self.ui.document.info.has_pages` with `if self.ui.paging` (and `if not...` with `if self.ui.rolling`.